### PR TITLE
Fix path to Dockerfile.ubuntu in Readme file instructions

### LINKED ISSUE #70: Fedora log when chose ubuntu
I cloned the latest head, and seeing fedora image pull instead of ubuntu

    ~/ipdk/build# ./ipdk install
    Loaded /root/ipdk/build/scripts/ipdk_default.env
    
    As last step execute:
      export PATH=/root/ipdk/build:$PATH
    to add the IPDK CLI to your path in the running environment!
    ~/ipdk/build# export PATH=/root/ipdk/build:$PATH
    ~/ipdk/build# BASE_IMG=ubuntu:20.04
    ~/ipdk/build# IMAGE_NAME=ipdk/p4-ovs-ubuntu20.04
    ~/ipdk/build# DOCKERFILE=${SCRIPT_DIR}/Dockerfile.ubuntu
    ~/ipdk/build# ipdk build --no-cache
    Loaded /root/ipdk/build/scripts/ipdk_default.env
    Loaded /root/.ipdk/ipdk.env
    ~/ipdk/build ~/ipdk/build
    Sending build context to Docker daemon  131.6kB
    Step 1/15 : ARG BASE_IMG=${BASE_IMG}
    Step 2/15 : FROM ${BASE_IMG} AS base
    33: Pulling from library/fedora
    c1323e24d83b: Extracting [============================================>      ]  55.15MB/61.52MB

Is this normal/ok? a typo? am I missing something?

### PR DESCRIPTION
**Pull request by asafravid**
No Scripts directory required in path, otherwise failure occurs

Error Details in -> Fixes #70

### CODE REVIEW

**Comment by stolsma** [comment]
I'm busy with a change to the cli so this is not needed anymore, anyhow https://github.com/ipdk-io/ipdk/blob/main/build/scripts/ipdk_default.env#L13 exactly describes what to do. For you removing $scripts will work but not if you use the ipdk command from another directory then build.

File: build/README_DOCKER.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ configuration file settings and inner workings!
 - The IPDK container is by default build with a `Fedora:33` base image. Build
   and run your IPDK container with the Ubuntu20.04 base image by setting
   `BASE_IMG=ubuntu:20.04`, `IMAGE_NAME=ipdk/p4-ovs-ubuntu20.04` and
-  `DOCKERFILE=${SCRIPT_DIR}/Dockerfile.ubuntu` in your user CLI configuration
+  `DOCKERFILE=Dockerfile.ubuntu` in your user CLI configuration
```
Comment (stolsma) [inline build/README_DOCKER.md:104]: This should be `${SCRIPT_DIR}/../Dockerfile` to let it work in other current directories then build.

**Review by stolsma — CHANGES_REQUESTED** [review]

**Comment by asafravid** [comment]
Good idea

File: build/README_DOCKER.md (hunk 1 above)
Comment (asafravid) [reply build/README_DOCKER.md:104]: Good Idea, will fix

**Comment by asafravid** [comment]
Replaced with PR https://github.com/ipdk-io/ipdk/pull/73

**Pull request closed without merging**